docs/confounding-variables.md

```diff
@@ -0,0 +1,40 @@
+# Confounding Variables
+
+A confounding variable is a factor that influences independent variable and dependant variable. 
```

Comment:
Please cite a scientific work that made effort for the definition (e.g. Szklo & Nieto, 2019, p.148 or others)